closed/get_j9_source.sh

```diff
@@ -67,7 +67,10 @@ git_urls[openj9]=https://github.com/eclipse-openj9/openj9
 git_urls[omr]=https://github.com/eclipse-openj9/openj9-omr
 
 currentbranch=$(git rev-parse --abbrev-ref HEAD)
-if [[ "$currentbranch" =~ v[0-9]+\.[0-9]+(\.[0-9]+)?-release ]] ; then
+if [[ "$currentbranch" =~ ibm-v[0-9]+\.[0-9]+(\.[0-9]+)?-release ]]; then
```

Comment:
We could just change the pattern so the prefix "ibm-" is optional, then I think `${currentbranch#ibm-}` will do the right thing in both cases.

Reply:
As in:
```
if [[ "$currentbranch" =~ (ibm-v)?[0-9]+\.[0-9]+(\.[0-9]+)?-release ]] ; then
	branches[openj9]=${currentbranch#ibm-}
	branches[omr]=${currentbranch#ibm-}
else
	branches[openj9]=master
	branches[omr]=openj9
fi
```
Please keep the space before the semicolon for consistency.

Reply:
The `v` is not optional. Shouldn't it be `(ibm-)?v`?

Reply:
Looks like it works
```
MY_BRANCH=ibm-v123
NEW_BRANCH=v123
$ echo ${MY_BRANCH#ibm-}
v123
$ echo ${NEW_BRANCH#ibm-}
v123
```

Reply:
Ah I didn't see the `BASH_REMATCH` comment. Do you want me to use that instead?

Reply:
I think using `BASH_REMATCH` would be more flexible: we could recognize other prefixes (or even suffixes) without needing to change `${BASH_REMATCH[2]}`.